### PR TITLE
[14.0][ADD] stock_picking_cancel_confirm

### PR DESCRIPTION
**Pull request by kittiu**
Add cancel reason dialog for stock picking.

### CODE REVIEW

**Comment by kittiu** [comment]
@ps-tubtim @rousseldenis

File: stock_picking_cancel_confirm/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright 2020 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import model
```
Comment (rousseldenis) [inline stock_picking_cancel_confirm/__init__.py]: Nitpicking but :

```suggestion
from . import models
```

File: stock_picking_cancel_confirm/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+When user click to cancel stock picking, a confirmation wizard will be show, with reason as optional.
```
Comment (rousseldenis) [inline stock_picking_cancel_confirm/readme/DESCRIPTION.rst]:
```suggestion
When user click to cancel stock picking, a confirmation wizard will be shown, with reason as optional.
```

Comment (kittiu) [reply stock_picking_cancel_confirm/readme/DESCRIPTION.rst]: Thanks @rousseldenis

**Review by rousseldenis — APPROVED** [review]
Code and func review

**Review by bodedra — APPROVED** [review]
Functional review. :+1:

**Pull request merged**